README.md

```diff
@@ -1,31 +1,69 @@
-# Reconstruction pipeline
-
-This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.
-It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB or used independently for modeling. 
+<h1 align="center">
+  <br>
+  <a href="http://www.amitmerchant.com/electron-markdownify"><img src="https://cdn.evbuc.com/eventlogos/50084740/tvbtaglinecrop.jpg" alt="The Virtual Brain" width="300"></a>
+  <br>
+  Reconstruction pipeline
+  <br>
+</h1>
+<h4 align="center">This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.</h4>
+<p align="center">
+  <a href="https://pegasus.isi.edu/downloads/">
+    <img src="https://img.shields.io/badge/Pegasus%20WMS-4.9.3-%2349A13D"
+         alt="Gitter">
+  </a>
+  <a href="https://neurostars.org/">
+	  <img src="https://img.shields.io/badge/Chat-on%20Neurostars-%23673B90">
+</a>
+  <a href="https://saythanks.io/">
+      <img src="https://img.shields.io/badge/SayThanks.io-%E2%98%BC-1EAEDB.svg">
+  </a>
+  <a href="https://www.paypal.com/">
+    <img src="https://img.shields.io/badge/%24-Donate-%239D2E2B">
+  </a>
+</p>
+
+<p align="center">
+  <a href="#Folder-structure">Folder structure</a> •
+  <a href="#Data-structure">Data structure</a> •
+  <a href="#How-to-launch">How to launch</a> •
+  <a href="#How-to-rerun">How to rerun</a> •
+  <a href="#Environment">Environment</a> •
+  <a href="#Contributors">Contributors</a>
+</p>
+<p>
+<hr>
+
+It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB or used independently for modeling.
 
 The mandatory inputs are DWI and T1 scans. Optionally, CT scans can be given as input, if sensors preprocessing is needed.
 
-We are using the [Pegasus WMS](https://pegasus.isi.edu/) in order to connect and automatize the pipeline steps. Pegasus is distributed under the Apache v2.0 license.
+We are using the <a href="https://pegasus.isi.edu/">Pegasus WMS</a> in order to connect and automatize the pipeline steps. Pegasus is distributed under the Apache v2.0 license.
```

Comment:
To be added:
`while our own code is GPL v3`

Reply:
Sure, ma'am.